packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -28,15 +41,59 @@ const normalizeContentValue = (value: string) => {
   return value;
 };
 
+/**
+ * Will merge all unconditional css expressions together.
+ *
+ * Input:
+ *
+ * ```
+ * [{ type: 'unconditional', css: 'color: blue' }, { type: 'unconditional', css: 'font-size: 20px' }]
+ * ```
+ *
+ * Output:
+ *
+ * ```
+ * [{ type: 'unconditional', css: 'color: blue; font-size: 20px' }]
+ * ```
+ *
+ * @param arr
+ */
+const reduceUnconditionalCssItems = (arr: Array<CssItem>): Array<CssItem> => {
```

Comment:
i feel this is the worst part of the implementation ATM - but it is what it is... 🤔 any thoughts?

Reply:
nit: description says merge and function name starts with reduce. Kinda confusing.

Reply:
> From what I understand from the implementation.

Might be a performance penalty but will clean the code:

- partition into two arrays using `filter` with check: `item.type === 'unconditional'`
- reduce `unconditional` array into an object `{ type: 'unconditional', css: 'color: blue; font-size: 20px' }` by concatenating `css`
- finally push this object into the other partitioned array
```ts
arrayWithoutUnconditionaItems.push({ type: 'unconditional', css: 'color: blue; font-size: 20px' })
```

Well not worse in terms of performance:
O(n + m) in first step
O(n) in second step
O(1) in last step

won't take much space either.

Reply:
ah yeah, that would be a more readable wouldn't it. i'll have a play